blockmanager.go

```diff
@@ -1144,35 +1120,23 @@ func (b *blockManager) handleInvMsg(imsg *invMsg) {
 // limitMap is a helper function for maps that require a maximum limit by
 // evicting a random rejected transaction if adding a new value would cause it
 // to overflow the maximum allowed.
-func (b *blockManager) limitMap(m map[wire.ShaHash]struct{}, limit int) error {
+func (b *blockManager) limitMap(m map[wire.ShaHash]struct{}, limit int) {
 	if len(m)+1 > limit {
-		// Generate a cryptographically random hash.
-		randHashBytes := make([]byte, wire.HashSize)
-		_, err := rand.Read(randHashBytes)
-		if err != nil {
-			return err
-		}
-		randHashNum := new(big.Int).SetBytes(randHashBytes)
-
-		// Try to find the first entry that is greater than the random
-		// hash.  Use the first entry (which is already pseudorandom due
-		// to Go's range statement over maps) as a fallback if none of
-		// the hashes in the map are larger than the random hash.
-		var foundHash *wire.ShaHash
+		// Remove a random entry from the map.  This relies on the fact
```

Comment:
I wouldn't say relies here since that contradicts what follows, which said that the iteration order does not matter at all due to the cryptographic hash.